data/borderlines.json

```diff
@@ -127,6 +127,57 @@
         }
       ]
     },
+    {
+      "name": "Syangja",
+      "latitude": 28.0197,
+      "longitude": 83.8049,
+      "timezone": "Asia/Kathmandu",
+      "officialLanguage": "Nepali",
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 1164,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 1261,
```

Comment:
I think the elevation value isn't correct for this?
What is the source of this data?

Reply:
corrected elevation, please review again. The source of the data is topographic-map.com